web/src/main/java/com/epam/rft/atsy/web/controllers/LoginController.java

```diff
@@ -11,15 +14,22 @@
 @Controller
 @RequestMapping("/login")
 public class LoginController {
-  private static final String VIEW_NAME = "login";
+  private static final String VIEW_NAME_LOGIN = "login";
+  private static final String VIEW_NAME_WELCOME = "secure/welcome";
 
   /**
    * Loads the page.
    * @return the ModelAndView object which describes the page
    */
   @RequestMapping(method = RequestMethod.GET)
   public ModelAndView pageLoad() {
-    return new ModelAndView(VIEW_NAME);
-  }
+    Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
 
-}
+    if (authentication != null && authentication.isAuthenticated()
+        && !(authentication instanceof AnonymousAuthenticationToken)) {
```

Comment:
This logical expression is a little bit complex. It can be a good idea to extract this logic to a private method with javadoc. This little refactor can improve readability and easier to understand.